packages/bpk-component-navigation-tab-group/src/BpkNavigationTabGroup.tsx

```diff
@@ -34,17 +34,21 @@ export type NavigationTabGroupTypes =
   (typeof NAVIGATION_TAB_GROUP_TYPES)[keyof typeof NAVIGATION_TAB_GROUP_TYPES];
 
 type TabItem = {
+  id: string;
   text: string;
   icon?: FunctionComponent<any> | null;
   href?: string;
+  dataCy?: string;
+  dataAnalyticsName?: string;
 };
 export type Props = {
+  id: string;
   tabs: TabItem[];
   type?: NavigationTabGroupTypes;
   /*
    * Index parameter to track which is clicked
    */
-  onItemClick: (tab: TabItem, index: number) => void;
+  onItemClick: (e: MouseEvent<HTMLButtonElement | HTMLAnchorElement>,tab: TabItem, index: number) => void;
```

Comment:
In GC header all click event and track be handled by handleClickAndTrack function that need MouseEvent ,so added args there
 https://github.skyscannertools.net/skyscanner/global-components/blob/e6dbbff3baa33e1a4a380dc2b27c10912cf8397e/src/header/utils/handleClickAndTrack.ts#L24